components/hab/src/command/studio/docker.rs

```diff
@@ -197,9 +201,7 @@ where
         cmd_args.push(vol.as_ref().into());
     }
 
-    if !is_serving_windows_containers(docker_cmd) {
```

Comment:
Was the removal of this check intentional?  Is this related to mounts too?

Reply:
since this PR would not enter this function when using windows containers, this check would always be `false`